README.md

```diff
@@ -29,6 +30,7 @@ const auth = oauth2({
       scope: ['user']
     }
   },
+  prefix: '/api/v1', 
```

Comment:
why? `const app = new Elysia({ prefix: '/api/v1' });` should do the job; otherwise its a bug with elysia

Reply:
Fixed.